packages/dnb-eufemia/src/extensions/forms/constants/countries.ts

```diff
@@ -2161,16 +2161,16 @@ const countries: Array<CountryType> = [
   },
   {
     i18n: {
-      en: 'United Kingdom',
-      nb: 'Storbritannia',
+      en: 'United Kingdom of Great Britain and Northern Ireland',
```

Comment:
```suggestion
      en: 'United Kingdom',
```

In the [UN list](https://unece.org/trade/cefact/unlocode-code-list-country-and-territory) it's only [United Kingdom](https://service.unece.org/trade/locode/gb.htm).

But then again, in the [ISO 3166 list](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes) its `United Kingdom of Great Britain and Northern Ireland`.

That's confusing.

Reply:
Yes it is a bit confusing. One might argue that if a user is sending money to North-Ireland they would take the closest name which is Ireland and thus submitting the wrong country name.

I think the most important part is just to have something we all agree on 🙂

Reply:
"United Kingdom" seems to be the widely recognized short form, while "United Kingdom of Great Britain and Northern Ireland" is the official long form used in formal and legal documents, especially in standards like ISO.

Based on this, I think we should opt for "United Kingdom".